### DISCUSSION TITLE
[灌水] 我和震暴者k22

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1098] [其他]
![mmexport1723300562384_edit_1956932066154517](https://github.com/user-attachments/assets/85606cb0-2043-47dd-9b2f-25db6d4114ad)
身为一名治安官，对我来说，每天的巡查都离不开这把枪。所以为了保证使用质量不会出问题，每天晚上睡觉前我基本上都会去保养我的武器，并且补充子弹。与前辈的三节棍不同，枪的保养还是很花时间的，加润滑油；清理残余火药；最后测试是否会卡膛，一套工序下来，基本上也要半个小时左右。

对我来说，武器的手感是很重要的，只有更熟练的使用武器才能打击更多的违法犯罪。

但放心，我绝对不会把震暴者指向良好市民的，所以还请大家遵纪守法…
![Screenshot_20240810_225655](https://github.com/user-attachments/assets/5a505f6b-14de-4b6d-9b99-21367387d448)

**Comment by mr99710** [comment]
治安官女士，我能要你的微信吗

**Reply by ZhuYuan-148** [reply]
![Screenshot_20240810_151823](https://github.com/user-attachments/assets/658f9d8e-3bbc-4eb1-ade1-7ce5ff80079a)

**Comment by Y-0930** [comment]
警长，我这有一批无封面录音带新货你收么[doge]

**Reply by ZhuYuan-148** [reply]
![Screenshot_20240810_151823](https://github.com/user-attachments/assets/504ec120-a683-477b-96f0-179e2e675ffb)

**Comment by zhangyj0523** [comment]
这位朱鸢警官是绳网的管理员吗，总能看到你，好高强度冲浪啊

**Reply by ZhuYuan-148** [reply]
周末上层给安排的假期，好无聊。在家顺便调查一下绳网就是了。
